main.py

```diff
@@ -1,10 +1,50 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as data:
+        player_data = json.load(data)
+
+    for player, data in player_data.items():
+        race = data["race"]
+        skills = race["skills"]
+        guild = data["guild"]
+
+        if not Race.objects.filter(name=race["name"]).exists():
```

Comment:
it will be better to use get_or_create() https://www.queworx.com/django/django-get_or_create/